src/app/components/keyboard/keyboard.component.html

```diff
@@ -28,15 +28,18 @@
         *ngFor="let key of thirdKeyboardRow"
         (click)="pressKeyValue(key)"
         [ngClass]="newBoxStyles(key)"
-        [disabled]="!this.roundFound || !this.nextRound"
+        [disabled]="!this.roundFound || !this.nextRound || this.round === 5"
       >
         <p>{{ key.letter }}</p>
       </button>
       <button
         class="col key"
         (click)="pressKeyValue(sendKey)"
         [disabled]="
-          !this.disableKeys || this.correctSyntaxWord || !this.nextRound
+          !this.disableKeys ||
+          this.correctSyntaxWord ||
+          !this.nextRound ||
+          this.round === 5
```

Comment:
```suggestion
          disableSendKey()
```
Try to put the less logic in the HTML the better